magefile.go

```diff
@@ -716,11 +725,6 @@ func (Test) packageOperatorIntegration(ctx context.Context, filter string) {
 	}
 	args = append(args, "./integration/package-operator/...")
 
-	_, isCI := os.LookupEnv("CI")
```

Comment:
Is that supposed to be removed?

Reply:
Yup. This was accidentally added with the intent to have code coverage on int tests but it doesn't do anything.